js/game.js

```diff
@@ -0,0 +1,87 @@
+/** @type {HTMLCanvasElement} */
+
+class Game{
+    constructor(ctx, width, height, player){
+        this.ctx = ctx;
+        this.width = width;
+        this.height = height;
+        this.player = player;
+        this.intervalId = null;
+        this.frames = 0;
+        this.enemies = [];
+        this.points = 0;
```

Comment:
One thing that you could've done is draw the road only when clicking the start button. For that, you would do this:

```
const road = new Image();
road.src = "../images/road.png"
this.imgRoad = road;
```

and then on the update function,

`this.ctx.drawImage(this.imgRoad, 0, 0, canvas.width, canvas.height)`